s2p/__init__.py

```diff
@@ -124,7 +124,8 @@ def pointing_correction(tile, i, cfg):
     rpc2 = cfg['images'][i]['rpcm']
 
     # correct pointing error
-    print('correcting pointing on tile {} {} pair {}...'.format(x, y, i))
+    if cfg["debug"]:
+        print('correcting pointing on tile {} {} pair {}...'.format(x, y, i))
```

Comment:
print or log statements? guessing this is print because of legacy and ownership reasons

Reply:
Keep m print. I want to keep the changes to upstream minimal.